src/main/java/com/simibubi/create/foundation/data/recipe/Mods.java

```diff
@@ -25,13 +26,33 @@ public enum Mods {
 	BOP("biomesoplenty"),
 	TF("twilightforest"),
 	ECO("ecologics"),
-
+	IC2("ic2", b -> b.reverseMetalPrefix()),
+	ATMO("atmospheric"),
```

Comment:
There already is an enum entry for atmospheric, please use that one instead.
Apart from that, looks good to me.

Reply:
Oh, sorry about that. I’ll fix that when I get a chance